msal/build.gradle

```diff
@@ -288,11 +288,13 @@ dependencies {
     androidTestImplementation "org.mockito:mockito-android:$rootProject.ext.mockitoAndroidVersion"
     // 'local' flavor dependencies
     localApi(project(":common")) {
-        transitive = false
+        transitive = true
+        exclude module: "jcip-annotations"
```

Comment:
This is being used by broker, but not MSAL. Why are we excluding it here?

Reply:
Removed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1380/commits/b58463de8199c9b744566d015d20583e1515badf